### ISSUE TITLE
50hz vs 60hz Playback

### ISSUE THREAD

**Issue by wbahnassi-if** [digital-sound-antiques/in_msx#9]
Currently the plug-in plays MSX music at the speed of an NTSC (60Hz) MSX machine. In the **Misc** page of the **Settings**, we have a **VSYNC Mode** option that allows choosing between PAL(50Hz) and NTSC(60Hz), but it doesn't seem to make any difference whether you choose one option or the other (unless I'm misunderstanding the option). I expected that choosing PAL(50Hz) will cause the music playback speed to slowdown to the rate of the European PAL MSX systems.

Thanks!

**Comment by okaxaki** [comment]
Please check the write permission is given to in_msx.ini. See Note section of the [README](https://github.com/digital-sound-antiques/in_msx).

Further, that PAL/NTSC option currently only works with MBM files.
(However, it will be nice that the option can be applied to the other formats...)

**Comment by wbahnassi-if** [comment]
Yes when changing the value in the UI I can see it being reflected in the in_msx.ini file correctly. Though I was trying it with KSS files. Is there any hope of seeing this option supported for KSS files too?

Thank you

**Comment by okaxaki** [comment]
Yes, I would consider to add override switch of PAL/NTSC mode on in_msx UI and libkss library next time I have time.

By the way, KSSX format itself has a PAL/NTSC flag inside the binary.
I fogot to document it out for a long long time...sorry.  
I have added the document about it [here](https://github.com/digital-sound-antiques/libkss/commit/37897fb8ab21cdce5a5e941685c6447b80b1d171).

So, if you can directly modify KSS binary, please set bit 6 of 0x0FH. 
This will result PAL frequency playback without waiting UI enhancement.

Thanks.

**Comment by wbahnassi-if** [comment]
Excellent. Yes changing bit 6 to 1 at 0x0FH drives the plugin to play the file at PAL speed. This will do until the plugin implements an "override" option to force KSS to play using PAL or NTSC timing regardless of the file's choice.

Thanks!
